app/assets/components/lks-header/template.njk

```diff
@@ -1,162 +1,163 @@
 {# Define some defaults #}
-{% set showNav = params.showNav if params.showNav else "false" %}
-{% set showSearch = params.showSearch if params.showSearch else "false" %}
+{% set showNav = true if params.showNav and params.primaryLinks else false %}
+{% set showSearch = true if params.showSearch else false %}
+{% set showValues = false %}
+{% set multiLogo = true if params.logos.scot or params.logos.wales or params.logos.ni and not params.logos.fpo else false %}
+{% set logoEn = true if params.logos.en else false %}
+
+{# {% if not multiLogo %}
+  <div class="nhsuk-header nhsuk-header--organisation nhsuk-header--parent" role="banner">
+    <div class="nhsuk-width-container nhsuk-header__container">
+      Part of <a href="https://www.hee.nhs.uk/">NHS Health Education England</a>
+    </div>
+  </div>
+{% endif %} #}
 
 <header class="nhsuk-header
-{%- if params.transactional or params.transactionalService %} nhsuk-header--transactional{% endif %}
-{%- if params.organisation and params.organisation.name %} nhsuk-header--organisation{% endif %}
-{%- if params.classes %} {{ params.classes }}{% endif %}" role="banner"
-{%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
+    {% if multiLogo %}nhsuk-header__multi{% endif %}
```

Comment:
Is it possible to move `nhsuk-header__multi` class down to `<div class="nhsuk-width-container nhsuk-header__container">` as it seems to be targeting styles for elements within this `div` ?

Reply:
Done

Reply:
Looks like `nhsuk-header__multi` class is still not moved in `<div class="nhsuk-width-container nhsuk-header__container">` (and still being rendered with `<header class="nhsuk-header nhsuk-header__multi nhsuk-header--organisation " role="banner">`). Could this be fixed ? thanks